packages/pds/.env.example

```diff
@@ -0,0 +1,24 @@
+# See more env options in src/config/env.ts
```

Comment:
Should we merge this file with the existing `packages/pds/example.dev.env`? which gets auto-copied for use with `make run-dev-pds`. I'd otherwise be worried this `.env.example` will be hard to find (default hidden dotfile) and get stale.

If we did that, would need to ensure that the config actually works for local dev, instead of sandbox. I think just commenting out partial strings would work.

Reply:
Yup yup that makes sense 👍 

i actually forgot we already had an .env.example

Reply:
alright I actually removed the `make run-dev-pds` & `make run-dev-bsky` commands as well as their env files

Right now, the best way to run these services in dev is in the dev-env the packages themselves aren't set up very well with service entry files. So I also moved over the `dotenv` code to dev-env so that it's env can be adjusted

I don't think we were using those Make commands at all during development, but lmk if this affects your ability to develop.

In the future, it might be worth taking another pass at refactoring running each of theses services in dev mode

Reply:
I do use those make commands fairly often, for the PDS and appview, distinct from `dev-env`. my workflow is basically to run `make run-dev-*` in a bunch of different terminals and a bunch of repos, while editing 1-2 code bases.

but I can try to get all that working in a follow-up PR, don't need to block this one on that side feature